worker/operations.py

```diff
@@ -143,48 +149,132 @@ def cosmos_extraction(document_id, filename, downloaded_document, force_run=Fals
         logger.info(
             f"Sending PDF to backend knowledge service with document id {document_id} at {cosmos_text_extraction_url}"
         )
-        response = requests.post(cosmos_text_extraction_url, files=put_payload, data=data_form)
+        response = requests.post(
+            cosmos_text_extraction_url, files=put_payload, data=data_form
+        )
         logger.info(
             f"Response received from backend knowledge service with status code: {response.status_code}"
         )
         extraction_json = response.json()
         logger.debug("COSMOS response object: %s", extraction_json)
         status_endpoint = extraction_json["status_endpoint"]
-        result_endpoint = f"{extraction_json['result_endpoint']}/text"
+        result_endpoint = f"{extraction_json['result_endpoint']}"
+        result_endpoint_text = f"{extraction_json['result_endpoint']}/text"
+        equations_endpoint = (
+            f"{extraction_json['result_endpoint']}/extractions/equations"
+        )
+        figures_endpoint = f"{extraction_json['result_endpoint']}/extractions/figures"
+        tables_endpoint = f"{extraction_json['result_endpoint']}/extractions/tables"
 
         job_done = False
 
         for i in range(MAX_ITERATIONS):
             status = requests.get(status_endpoint)
             status_data = status.json()
-            logger.info("Polled status endpoint %s times:\n%s", i+1, status_data)
-            job_done = status_data['error'] or status_data['job_completed']
+            logger.info("Polled status endpoint %s times:\n%s", i + 1, status_data)
+            job_done = status_data["error"] or status_data["job_completed"]
             if job_done:
                 break
             time.sleep(POLLING_INTERVAL)
 
         if not job_done:
             logger.error("ERROR: Job not complete after %s seconds.", MAX_EXECTION_TIME)
             raise Exception(f"Job not complete after {MAX_EXECTION_TIME} seconds.")
-        elif status_data['error']:
+        elif status_data["error"]:
             logger.error("An unexpected error occurred: %s", {status_data["error"]})
-            raise Exception(f"An error occurred when processing in Cosmos: {status_data['error']}")
+            raise Exception(
+                f"An error occurred when processing in Cosmos: {status_data['error']}"
+            )
+
+        logger.info(result_endpoint_text)
```

Comment:
This is not a great `info` level log since it's just printing the endpoint. `"Obtaining extractions from Cosmos at: {result_endpoint_text}"` etc would be preferable